index.js

```diff
@@ -300,5 +335,6 @@ module.exports = {
     removeVersion,
     tagTemplate,
     removeTag,
-    getVersionFromTag
+    getVersionFromTag,
+    validatePipelineTemplate
```

Comment:
We need to add unit tests
Refer https://github.com/screwdriver-cd/template-main/blob/master/test/index.test.js#L70

Reply:
Per discussion, will add tests for new commands in a follow up PR